packages/beacon-node/src/chain/chain.ts

```diff
@@ -156,6 +157,12 @@ export class BeaconChain implements IBeaconChain {
 
     const signal = this.abortController.signal;
     const emitter = new ChainEventEmitter();
+
+    // We set infinity to prevent MaxListenersExceededWarning which get logged when listeners > 10
+    // Since it is perfectly fine to have listeners > 10
+    emitter.setMaxListeners(Infinity);
+    setMaxListeners(Infinity, signal);
```

Comment:
Could you show the stack trace proving that this very specific event emitter gets more that 10 listeners?

Reply:
I could not create a situation where the warning is triggered locally. So I have removed the changes.